cloud/providers/linode/apply.go

```diff
@@ -178,6 +178,9 @@ func (cm *ClusterManager) applyCreate(dryRun bool) (acts []api.Action, err error
 			if err = CreateCredentialSecret(cm.ctx, kc, cm.cluster); err != nil {
 				return
 			}
+			if err = createCredentialSecret(cm.ctx, kc, cm.cluster); err != nil {
```

Comment:
Make a comment why this is needed? . function name may be createLinodeCloudSecret